index.html

```diff
@@ -1,15 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Awesome Bookstore</title>
   <link rel="stylesheet" href="index.css" />
-  <title>Awesome books with ES6</title>
 </head>
+
 <body>
+  <header>
+    <nav class="nav-bar">
+      <div class="logo">
+        <h2>Awesome Books</h2>
+      </div>
+      <div class="nav-list-container">
+        <ul class="nav-list">
+          <li class="nav-list-item book-page"><a href="#display-Books">List <span class="divider">|</span></li></a>
+          <li class="nav-list-item add-page"><a href="# add-New">Add new <span class="divider">|</span></li></a>
+          <li class="nav-list-item info-page"><a href="#contact">Contact <span class="divider"></span></li></a>
+
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <p class="date"></p>
```

Comment:
- [ ] As requested by the previous code reviewer, From the project requirements, the current date is displayed below the navigation bar, currently, your date is not displayed, please fix this :pray: 
![image](https://user-images.githubusercontent.com/92303891/205781799-8b1f8973-2daa-4f59-baec-0e22d3e82f07.png)